docker-compose.yml

```diff
@@ -43,3 +44,16 @@ services:
       - pubsub
       - start
       - "--host-port=0.0.0.0:8793"
+
+  kafka:
+    image: wurstmeister/kafka
```

Comment:
Use the "more official" image from Kafka. I would propose to use https://hub.docker.com/r/confluent/kafka/ and also stick image version it should supposed to work. Test may fail in the future when new version with breaking change is introduced. Same with zookeeper.